input/resources/StructureDefinition-profile-SignalUSCoreOrganizationProfile.json

```diff
@@ -30,6 +30,11 @@
         "path": "Organization.extension.extension",
         "sliceName": "code"
       },
+      {
+        "id": "Organization.extension:qualification.extension:code.url",
```

Comment:
this doesn't belong here, it should be in the Qualification PR.

Reply:
Removed the qualification extension, will get that added as part of other issue.

#https://github.com/enjoysparkling/signal-mso-fhir-profiles/pull/113/commits/3046f47191ea43f6485961f3ad1b7dba01077e0e